src/components/Books.js

```diff
@@ -1,14 +1,23 @@
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { fetchBooks, getStatus } from '../redux/books/bookSlice';
 import Book from './Book';
 import BookForm from './BookForm';
 
 const Books = () => {
-  const books = useSelector((state) => state.books);
+  const dispatch = useDispatch();
+  const books = useSelector((state) => state.books.list);
+  const status = useSelector(getStatus);
```

Comment:
[OPTIONAL].
- [ ] I do not see any logical meaning of this `status = "idle"` here, even if absent, the code would work properly, kindly it would be great to not over-engineer and use only the `KISS(Keep It Simple and Smart)` method of clean code.

Reply:
@christianbiring1 , hi. thank you for this feedback. i will make the suggested changes in my next PR